contracts/L2ForkArbitrator.sol

```diff
@@ -114,22 +141,29 @@ contract L2ForkArbitrator is IBridgeMessageReceiver {
     /// @dev Talks to the L1 ForkingManager asynchronously, and may fail.
     /// @param question_id The question in question
     function requestActivateFork(bytes32 question_id) public {
-        require(!isForkInProgress, "Already forking"); // Forking over something else
+        if (isForkInProgress) {
+            revert ForkInProgress(); // Forking over something else
+        }
 
         RequestStatus status = arbitrationRequests[question_id].status;
-        require(
-            status == RequestStatus.QUEUED ||
```

Comment:
Was || now && ?

Reply:
status != RequestStatus.QUEUED || status != RequestStatus.FORK_REQUEST_FAILED

would always return true, right?

Since we need to invert == || ==, I think its correct to use != && !=